core/eolearn/core/eodata_merge.py

```diff
@@ -31,7 +31,7 @@
 OperationInputType = Union[Literal[None, "concatenate", "min", "max", "mean", "median"], Callable]
 
 
-def merge_eopatches(
+def merge_eopatch_content(
```

Comment:
Should this become a "private" method in `eodata.py`?

Reply:
i guess it could, we can just silence pylint when it complains about importing

Reply:
done